tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts_full.py

```diff
@@ -338,12 +337,8 @@ def fail_copy(src, dst):
         assert file1.read_text() == "hello"
 
 
-@pytest.mark.wandb_core_failure(feature="artifacts")
-def test_check_changed_artifact_then_download(wandb_init, tmp_path, monkeypatch):
+def test_check_changed_artifact_then_download(wandb_init, tmp_path):
     """*Do* re-download an artifact if it's been modified in place."""
-    cache_dir = tmp_path / "cache"
-    monkeypatch.setenv("WANDB_CACHE_DIR", str(cache_dir))
```

Comment:
`ArtifactsCache` is a singleton so this may or may not have an effect. It's not needed anyway, so I removed it.